src/pug/characters/nene.pug

```diff
@@ -3,8 +3,8 @@
   color: '#dd95c6',
   title: {
     image: "https://vignette.wikia.nocookie.net/new-game/images/c/ce/Th-c7.png/revision/latest?cb=20180121131923",
-    name: "Nenecci",
-    quote: '"Nene quote here"'
+    name: "God",
```

Comment:
I believe you may have made a typo here, please update

Reply:
fuck you xetera your mom doesn't love you

Reply:
im sorry i didnt mean it